icons/submenu-arrow.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 465 1024"><path d="M465.455 525.838L76.738 1020.226.001 966.133l319.528-455.391L.001 54.093 76.738 0l388.717 491.872z"></path></svg>
```

Comment:
I have the same as `chevron-right.svg` in https://github.com/hlxsites/netcentric/pull/47/files#diff-4de610087b2ff237a938415695ab7bdbcc8e0484e94fdc127e6fb59c2eba6d65

can you rename it so we can use it multi-purpose?